samples/batch/transcription-enabled-storage/Connector/LanguageIdentification.cs

```diff
@@ -0,0 +1,103 @@
+﻿// <copyright file="LanguageIdentification.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CognitiveServices.Speech;
+    using Microsoft.CognitiveServices.Speech.Audio;
+
+    public class LanguageIdentification
```

Comment:
What is this used for? It seems use online transcription but not batch?

Reply:
yeah - the only reason it uses online transcription is because Language Id is not supported by batch. 

However, batch transcription is still needed to avoid long running azure function